searchcore/src/vespa/searchcore/proton/attribute/attribute_collection_spec_factory.cpp

```diff
@@ -35,6 +37,7 @@ AttributeCollectionSpecFactory::create(const AttributesConfig &attrCfg,
         }
         grow.setDocsGrowDelta(grow.getDocsGrowDelta() + skew);
         cfg.setGrowStrategy(grow);
+        cfg.setCompactionStrategy(_compaction_strategy);
```

Comment:
Do we have any existing tests on how we create the attribute collection spec config? If so, that should be extended. If not, we should consider adding tests for this.